tpkg/cutest/cutest_namedb.c

```diff
@@ -824,27 +811,20 @@ static void
 test_add_del_2(CuTest *tc, namedb_type* db)
 {
 	zone_type* zone = find_zone(db, "example.org");
-	udb_ptr udbz;
-	if(!udb_zone_search(db->udb, &udbz,
-		dname_name(domain_dname(zone->apex)),
-		domain_dname(zone->apex)->name_size)) {
-		printf("cannot find udbzone\n");
-		exit(1);
-	}
 	check_namedb(tc, db);
 	zone->is_ok = 0;
 
-	del_str(db, zone, &udbz, "example.org. IN SOA ns.example.org. hostmaster.example.org. 2011041200 28800 7200 604800 3600\n");
+	del_str(db, zone, "example.org. IN SOA ns.example.org. hostmaster.example.org. 2011041200 28800 7200 604800 3600\n");
 	check_namedb(tc, db);
-	del_str(db, zone, &udbz, "example.org. IN NS ns.example.com.\n");
+	del_str(db, zone, "example.org. IN NS ns.example.com.\n");
 	check_namedb(tc, db);
-	del_str(db, zone, &udbz, "example.org. IN NS ns2.example.com.\n");
+	del_str(db, zone, "example.org. IN NS ns2.example.com.\n");
 	check_namedb(tc, db);
 	/* the root has not been deleted */
 	CuAssertTrue(tc, domain_table_count(db->domains) != 0);
 	CuAssertTrue(tc, db->domains->root && db->domains->root->number);
 
-	udb_ptr_unlink(&udbz, db->udb);
+//	udb_ptr_unlink(&udbz, db->udb);
```

Comment:
I think this line and the empty line above can be removed.
```suggestion
```

Reply:
Fixed.